includes/classes/order.php

```diff
@@ -114,6 +114,7 @@ protected function query($order_id = 0) {
                         'last_modified' => $order->fields['last_modified'],
                         'ip_address' => $order->fields['ip_address'],
                         'order_weight' => $order->fields['order_weight'],
+                        'language_id' => $order->fields['language_id']
```

Comment:
Hmmm .... given the suggestions to use `$lng->get_language_data_by_id($id)` earlier, I wonder if it makes sense to set this once into the `order` object ... something like:

```php
$this->info = .... 'language'=>$lng->get_language_data_by_id($order->fields['language_id']),
```
and similarly in the `cart` function too.

Just would have to be careful when writing to the db that we actually store only the `int` of `language_id`;

I'm undecided whether to put both `language` (array) and `language_id` (int) into the `info` array. It wouldn't strictly be needed, as one could easily get it via `$order->info['language']['id']`.

The benefit to this is that we have access to `id`, `directory`, `name`, `code`, and even `icon` as sub-elements of the `['language']` element.


Would it also make sense to add `localized_orders_status_name` directly here (well, up around line `111`), instead of deferring that to the admin `orders.php` app?